[2025-03-15T03:03:59.991+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T03:03:59.997+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inference-pipeline.inference manual__2025-03-15T03:03:55.538528+00:00 [queued]>
[2025-03-15T03:03:59.999+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inference-pipeline.inference manual__2025-03-15T03:03:55.538528+00:00 [queued]>
[2025-03-15T03:03:59.999+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-15T03:04:00.003+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): inference> on 2025-03-15 03:03:55.538528+00:00
[2025-03-15T03:04:00.010+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=140) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T03:04:00.011+0000] {standard_task_runner.py:72} INFO - Started process 150 to run task
[2025-03-15T03:04:00.011+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'inference-pipeline', 'inference', 'manual__2025-03-15T03:03:55.538528+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/inference_pipeline.py', '--cfg-path', '/tmp/tmpsh711ljz']
[2025-03-15T03:04:00.013+0000] {standard_task_runner.py:105} INFO - Job 56: Subtask inference
[2025-03-15T03:04:00.031+0000] {task_command.py:467} INFO - Running <TaskInstance: inference-pipeline.inference manual__2025-03-15T03:03:55.538528+00:00 [running]> on host eb8bcabe4d60
[2025-03-15T03:04:00.057+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='user' AIRFLOW_CTX_DAG_ID='inference-pipeline' AIRFLOW_CTX_TASK_ID='inference' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T03:03:55.538528+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T03:03:55.538528+00:00'
[2025-03-15T03:04:00.057+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-15T03:04:00.111+0000] {inference_step.py:54} INFO - Models in the model registry: []
[2025-03-15T03:04:00.111+0000] {inference_step.py:65} WARNING - No model in the model registry under the name: cancellation_estimator.
[2025-03-15T03:04:00.127+0000] {inference_step.py:73} INFO - Batch columns: Index(['lead_time', 'arrival_date_year', 'arrival_date_week_number',
       'arrival_date_day_of_month', 'stays_in_weekend_nights',
       'stays_in_week_nights', 'adults', 'children', 'babies',
       'is_repeated_guest', 'previous_cancellations',
       'previous_bookings_not_canceled', 'days_in_waiting_list', 'adr',
       'required_car_parking_spaces', 'total_of_special_requests',
       'arrival_date_month', 'meal', 'market_segment', 'distribution_channel',
       'reserved_room_type', 'assigned_room_type', 'customer_type', 'country',
       'booking_changes', 'agent', 'company'],
      dtype='object')
[2025-03-15T03:04:00.128+0000] {inference_step.py:36} WARNING - No model used for prediction. Model registry probably empty.
[2025-03-15T03:04:00.128+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-15T03:04:00.132+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-15T03:04:00.132+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=inference-pipeline, task_id=inference, run_id=manual__2025-03-15T03:03:55.538528+00:00, execution_date=20250315T030355, start_date=20250315T030359, end_date=20250315T030400
[2025-03-15T03:04:00.155+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T03:04:00.161+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-15T03:04:00.162+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
